### PR TITLE
Fix #68. FAQ: block modifier which affects elements

### LINKED ISSUE #68: Block modifiers that effect elements
I'm not sure if & where this falls into the BEM methodology, but I'm interested in where this might fall.

If I have a block modifier, for example "xmas," and I want the elements within that block to also be xmas themed, how would it be best to do it.

Does a `--xmas` suffix for every element seem necessary? Or would this be the one use-case for nesting (e.g. `block--xmas block__elem { ... }`?

### PR DESCRIPTION
**Pull request by varya**

### CODE REVIEW

File: src/markdown/faq.md [hunk 1]
```diff
@@ -75,6 +75,32 @@ In case of combining 2 or more modifiers at the same block `<div class="block--t
 
 When operating modifiers dynamically with JavaScript, additional modifier is more handy. Switching it off would mean only removing one CSS class from the DOM node with no need to add the core block CSS class back as it sits there forever.
 
+<a id="block-modifier-affects-elements"></a>
+## Can a block modifier affect elements?
+
+> If I have a block modifier, for example "xmas," and I want the elements within that block to also be xmas themed, how
```
Comment (floatdrop) [inline src/markdown/faq.md]: `"xmas,"` -> `"xmas",`

**Comment by floatdrop** [comment]
:ok:

**Comment by varya** [comment]
@floatdrop Typo fixed

**Comment by floatdrop** [comment]
:+1:

**Pull request merged**